packages/template-retail-react-app/app/components/forms/login-fields.jsx

```diff
@@ -6,26 +6,52 @@
  */
 import React from 'react'
 import PropTypes from 'prop-types'
-import {Stack} from '@salesforce/retail-react-app/app/components/shared/ui'
-import useLoginFields from '@salesforce/retail-react-app/app/components/forms/useLoginFields'
+import {FormattedMessage} from 'react-intl'
+import {Stack, Box, Button} from '@salesforce/retail-react-app/app/components/shared/ui'
 import Field from '@salesforce/retail-react-app/app/components/field'
+import useLoginFields from '@salesforce/retail-react-app/app/components/forms/useLoginFields'
+import {noop} from '@salesforce/retail-react-app/app/utils/utils'
 
-const LoginFields = ({form, prefix = ''}) => {
+const LoginFields = ({
+    form,
+    clickForgotPassword = noop,
+    prefix = '',
+    hideEmail = false,
+    hidePassword = false
+}) => {
     const fields = useLoginFields({form, prefix})
     return (
         <Stack spacing={5}>
-            <Field {...fields.email} />
-            <Field {...fields.password} />
+            {!hideEmail && <Field {...fields.email} />}
+            {!hidePassword && (
+                <Stack>
+                    <Field {...fields.password} />
+                    <Box>
+                        <Button variant="link" size="sm" onClick={clickForgotPassword}>
+                            <FormattedMessage
+                                defaultMessage="Forgot password?"
+                                id="login_form.link.forgot_password"
+                            />
+                        </Button>
+                    </Box>
```

Comment:
Would you consider not showing the forgot password link if no handler is passed in? You might have to re-evaluate setting it's default value to a noop. But I don't see a point to showing the link if there is no handler.

That being said, are we intending this to be a breaking change? I see that you are doing a bit of refactoring with regards to files moving around and stuff.

Reply:
Are there cases of PWA Kit implementations where a `clickForgotPassword` handler is not passed? I was thinking that all implementations should include a password reset link as it seems like all login forms nowadays have this option.

Reply:
Yes as this is changing the UI this feature will be considered a breaking change

Reply:
The comment about not showing the link if no function is passed in is a general comment about how developers might use the component. We don't know if someone else is going to use that component in another page or dialog or whatever. So why would be show the link if there is no actionable event to take if someone clicks it?

Reply:
Makes sense! Removed `clickForgotPassword` to default to `noop` and only show the link when it is defined

Reply:
Based on the current code base there is a pattern to name functions that respond to user actions starting with 'handle'... in this case please change it to `handleForgotPasswordClick'